backend/src/main/java/com/htecgroup/skynest/controller/BucketsController.java

```diff
@@ -0,0 +1,105 @@
+package com.htecgroup.skynest.controller;
+
+import com.htecgroup.skynest.model.response.BucketResponse;
+import com.htecgroup.skynest.model.response.ErrorMessage;
+import com.htecgroup.skynest.model.response.UserResponse;
+import com.htecgroup.skynest.service.BucketService;
+import io.swagger.v3.oas.annotations.Operation;
+import io.swagger.v3.oas.annotations.media.Content;
+import io.swagger.v3.oas.annotations.media.ExampleObject;
+import io.swagger.v3.oas.annotations.media.Schema;
+import io.swagger.v3.oas.annotations.responses.ApiResponse;
+import io.swagger.v3.oas.annotations.responses.ApiResponses;
+import io.swagger.v3.oas.annotations.tags.Tag;
+import lombok.AllArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/buckets")
+@AllArgsConstructor
+@Log4j2
+@Tag(name = "Bucket API", description = "Bucket-related operations")
+public class BucketsController {
+
+  private BucketService bucketService;
+
+  @Operation(summary = "Get bucket with given uuid")
+  @ApiResponses(
+      value = {
+        @ApiResponse(
+            responseCode = "200",
+            description = "Bucket returned",
+            content = {
+              @Content(
+                  mediaType = "application/json",
+                  schema = @Schema(implementation = UserResponse.class),
+                  examples = {
+                    @ExampleObject(
```

Comment:
Is this wrong example object for status code 200? Should be BucketResponse.class and corresponding example. Not UserResponse example.

Reply:
Yes, you're right I forgot to change the example after copying it from the other class.